applications/nrf5340_audio/src/bluetooth/le_audio_cis_headset.c

```diff
@@ -248,6 +324,8 @@ static void disconnected_cb(struct bt_conn *conn, uint8_t reason)
 
 	bt_conn_unref(default_conn);
 	default_conn = NULL;
+
+	advertising_start();
```

Comment:
In the before, once if disconnection happened then the host will start advertising automatically.
Would this mechanism changed after we involved the bond mechanism?

Reply:
Ah... we are using BT_LE_ADV_OPT_ONE_TIME now, so we have to restart the ADV manually.

Reply:
And that is because we want to change the way it advertises depending on if it is bonded or not